### ISSUE TITLE
From "Project" details, offer the option to connect a new item

### ISSUE THREAD

**Issue by maximelehericy** [nextcloud/server#33100]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Is your feature request related to a problem? Please describe.**
Problem is, when I want to add a new item (folders, files, conversations, boards) to an existing project, I must first go to the new item, and then add it to the project.

Path goes : **ITEM >>> PROJECT** (you have to navigate to the item, and from the item, connect it to the project)

**Describe the solution you'd like**
My mind usually do it the other way around, **PROJECT >>> ITEM**.
In other words, I'd like to be able, from an existing project, to add a new item (exactly the same way as at the project creation when you first link two items).

Add to project details the opportunity to connect a new item.
![image](https://user-images.githubusercontent.com/87175513/177149572-439e9e20-db58-4519-8bc6-6c60f4ae2286.png)
